src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -491,16 +490,16 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
           if (eventType === 'query') {
             return callback({
               timestamp: event.timestamp,
-              query: fields.query,
-              params: fields.params,
-              duration: fields.duration_ms,
+              query: fields?.query ?? event.query,
```

Comment:
Clean this up

Reply:
If it works, it works

Reply:
fields is only defined with NodeEngine and event with napi?

Reply:
Correct, the logging is different depending on the engine